3scale/3scale-operator
# Implemented skipping apicast service port reconcile based on annotation

This will resolve [THREESCALE-8385](https://issues.redhat.com/browse/THREESCALE-8385)

### Verification
- Deploy 3scale
- Add the annotation to the APIManager `marin3r.3scale.net/disable-apicast-service-port-reconcile: "true"`
- Confirm apicast service gateway port is 8443
- Made any change to apicast-production/staging Deployment Config to trigger reconciliation, like add a random annotation value `test-annotation: 'value'`
- Check apicast service that the gateway port stayed 8443 and hasn't changed to 8080

**Note**: _The above steps should be repeated with the annotation set to `'false'` or no annotation present at all and check that port changes upon reconciliation._
